### PR TITLE
added date_created to: ags, datasets, events, indicators, metrics,sto…

### PR DESCRIPTION
**Pull request by cueneythizli**
added date_created to: ags, datasets, events, indicators, metrics,stories

https://github.com/policycompass/policycompass/issues/676

### CODE REVIEW

**Comment by slomo** [comment]
I think we need to update some more fixtures and rest viewsets as well.

**Comment by slomo** [comment]
I am currently updating all the fixtues

File: apps/eventsmanager/models.py [hunk 1]
```diff
@@ -15,9 +15,8 @@ class Event(models.Model):
     userID = models.IntegerField()
     externalResourceID = models.IntegerField(blank=True, default=0)
     dateAddedToPC = models.DateTimeField(auto_now_add=True)
-    dateIssuedByExternalResource = models.DateTimeField(auto_now_add=True)
-    dateModified = models.DateTimeField(auto_now_add=True)
-    viewsCount = models.IntegerField(blank=True)
```
Comment (FabiApfelkern) [inline apps/eventsmanager/models.py]: I don't know. It's an old field. I guess it can be deleted.

**Comment by FabiApfelkern** [comment]
I also updated some fixture already. See the master.

**Pull request merged**